apps/dolly-frontend/src/main/js/src/components/fagsystem/pdlf/form/partials/kontaktinformasjonForDoedsbo/KontaktinformasjonForDoedsbo.js

```diff
@@ -7,34 +7,48 @@ import { Adresse } from './Adresse'
 import { FormikDollyFieldArray } from '~/components/ui/form/fieldArray/DollyFieldArray'
 import { AvansertForm } from '~/components/fagsystem/pdlf/form/partials/avansert/AvansertForm'
 import { initialKontaktinfoForDoedebo } from '~/components/fagsystem/pdlf/form/initialValues'
+import Panel from '~/components/ui/panel/Panel'
+import { erForste, panelError } from '~/components/ui/form/formUtils'
+import { Vis } from '~/components/bestillingsveileder/VisAttributt'
+
+const doedsboAttributt = 'pdldata.person.kontaktinformasjonForDoedsbo'
 
 export const KontaktinformasjonForDoedsbo = ({ formikBag }) => {
 	return (
-		<FormikDollyFieldArray
-			name="pdldata.person.kontaktinformasjonForDoedsbo"
-			header="Kontaktinformasjon for dødsbo"
-			newEntry={initialKontaktinfoForDoedebo}
-			canBeEmpty={false}
-		>
-			{(path, idx) => {
-				return (
-					<>
-						<FormikSelect
-							name={`${path}.skifteform`}
-							label="Skifteform"
-							options={Options('skifteform')}
-							isClearable={false}
-						/>
-						<FormikDatepicker
-							name={`${path}.attestutstedelsesdato`}
-							label="Utstedelsesdato skifteattest"
-						/>
-						<Kontakt formikBag={formikBag} path={path} />
-						<Adresse formikBag={formikBag} path={`${path}.adresse`} />
-						<AvansertForm path={path} kanVelgeMaster={false} />
-					</>
-				)
-			}}
-		</FormikDollyFieldArray>
+		<Vis attributt={doedsboAttributt}>
+			<Panel
+				heading="Kontaktinformasjon for dødsbo"
+				hasErrors={panelError(formikBag, doedsboAttributt)}
+				iconType="doedsbo"
+				startOpen={() => erForste(formikBag.values, [doedsboAttributt])}
+			>
+				<FormikDollyFieldArray
+					name="pdldata.person.kontaktinformasjonForDoedsbo"
+					header="Kontaktinformasjon for dødsbo"
+					newEntry={initialKontaktinfoForDoedebo}
+					canBeEmpty={false}
+				>
+					{(path, idx) => {
+						return (
+							<>
+								<FormikSelect
+									name={`${path}.skifteform`}
+									label="Skifteform"
+									options={Options('skifteform')}
+									isClearable={false}
+								/>
+								<FormikDatepicker
+									name={`${path}.attestutstedelsesdato`}
+									label="Utstedelsesdato skifteattest"
+								/>
+								<Kontakt formikBag={formikBag} path={path} />
```

Comment:
La ikke merke til det før jeg testet nå, men denne gjør også det samme som nevnt over, at dersom man endrer kontakttype mellom advokat og organisasjon så gjør den et nytt kall for å hente inn samme organisasjoner hver gang 😅  Det er ikke noe som er blitt endret i PRen nå, men hadde vært digg å ha den i en useEffect så det bare hentes en gang hvis du skulle ha lyst 😄